lib/identity_config.rb

```diff
@@ -257,6 +257,7 @@ def self.build_store(config_map)
     config.add(:in_person_outage_expected_update_date, type: :string)
     config.add(:in_person_outage_message_enabled, type: :boolean)
     config.add(:in_person_proofing_enabled, type: :boolean)
+    config.add(:in_person_proofing_enforce_tmx, type: :boolean)
```

Comment:
I've gotta hurry and get my PR merged since we're both using this. 😅

Reply:
i just added it here temporarily so I could test